Slim/Plugin/Podcast/Parser.pm

```diff
@@ -1,8 +1,11 @@
 package Slim::Plugin::Podcast::Parser;
 
+use strict;
+
 use Date::Parse qw(strptime str2time);
 use Scalar::Util qw(blessed);
 use URI;
+use XML::Simple;
```

Comment:
This is not really required here, is it?

Reply:
no